spec/models/case_cycle_spec.rb

```diff
@@ -94,8 +94,14 @@
 #  submitted_on    :date             not null
 #  created_at      :datetime         not null
 #  updated_at      :datetime         not null
+#  user_id         :uuid             not null
```

Comment:
Need to add an example (test) for the `belongs_to` relationship, e.g.
```
it { should belong_to(:user) } # or it { is_expected_to belong_to(:user) }
```

Reply:
Concur!